be/src/common/config.h

```diff
@@ -936,9 +936,12 @@ CONF_mInt32(orc_writer_version, "-1");
 // parquet reader
 CONF_mBool(parquet_coalesce_read_enable, "true");
 CONF_Bool(parquet_late_materialization_enable, "true");
-CONF_Bool(parquet_page_index_enable, "true");
+CONF_mBool(parquet_page_index_enable, "true");
```

Comment:
Why do we need to add this switch if it's already released and enabled by default?